src/Microsoft.AspNet.Razor/Chunks/Generators/TagHelperDirectiveChunkGenerator.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Parser.SyntaxTree;
+
+namespace Microsoft.AspNet.Razor.Chunks.Generators
+{
+    public abstract class TagHelperDirectiveChunkGenerator : SpanChunkGenerator
```

Comment:
The method doesn't add value especially since the `Span` is available. This code doesn't need these complications and an `interface` is definitely not required.